pennylane/plugins/default_gaussian.py

```diff
@@ -841,7 +841,7 @@ def reduced_state(self, wires):
         r""" Returns the vector of means and the covariance matrix of the specified wires.
 
         Args:
-            wires (int of Sequence[int]): indices of the requested wires
+            wires (Iterable[int] or int): indices of the requested wires # TODO: re-assess for nonconsec wires
```

Comment:
Probably best not to have the comment in the docstring 😆

Reply:
You are right. I deleted it altogether, I will remember this tag anyways.